### PR TITLE
Enable automation of NSTableView/NSOutlineView

### PR DESCRIPTION
**Pull request by manish**

### CODE REVIEW

**Comment by manish** [comment]
build

**Comment by manish** [comment]
I have built the dmg locally and installed it. Then I updated my UITests to use this build and everything works fine.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Components.AutoTest.Results/NSTableResult.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
+﻿//
+// NSTableResult.cs
+//
+// Author:
+//       Manish Sinha <manish.sinha@xamarin.com>
+//
+// Copyright (c) 2016 Xamarin Inc.
```
Comment (iainx) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components.AutoTest.Results/NSTableResult.cs]: Microsoft? :trollface:

**Comment by iainx** [comment]
LGTM, feel free to commit once the conflicts are fixed

**Comment by manish** [comment]
Looks like there is an issue with the second commit in this PR. Fixing it

**Comment by Therzok** [comment]
LGTM.

**Pull request closed without merging**